features/step_definitions/kata.rb

```diff
@@ -227,3 +227,189 @@
     end
   end
 end
+
+Given /^I run must-gather command$/ do
+  require 'json'
+  command = 'quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-must-gather:202106221012'
+  logger.info("==================================TEST IS RUNNING=========================================")
+  logger.info("Running must-gather command")
+  @result = env.admin.cli_exec(:oadm_must_gather, image:command)
+  pods_json = ''
+  @error_message = {}
+  dc_output = @result[:stdout]
+  @crio_logs_counter = 0
+  @audit_logs_counter = 0
+  logger.info("\nMust-gather image is:#{command}")
+  if dc_output.to_s.include? "logs_crio"
+    logger.info("CRI-O logs are in must-gather bundle")
+    dc_output.each_line do |line|
+      if line.to_s.include? "logs_crio"
+        #logger.info(line)
+        @crio_logs_counter += 1
+      end
+    end
+  end
+  logger.info("There are #{@crio_logs_counter} CRI-O logs in must-gather bundle")
+  logger.info("#{@crio_logs_counter} CRI-O logs matches 6 node cluster")
+  if dc_output.to_s.include? "audit.log"
+    logger.info("Audit logs are in must-gather bundle")
+    dc_output.each_line do |line|
+      if line.to_s.include? "audit.log"
+        #logger.info(line)
+        @audit_logs_counter += 1
+      end
+    end
+  end
+  logger.info("There are #{@audit_logs_counter} Audit logs in must-gather bundle")
+  logger.info("#{@audit_logs_counter} Audit logs matches 3 audit logs for every worker node")
+  raise "Failed to run must-gather command" unless @result[:success]
+end
+
+Given /^Pre-test checks$/ do
+  require 'json'
+  #require '/home/valiev/mygit/verification-tests/lib/world'
+  #require '/home/valiev/mygit/verification-tests/features/step_definitions/cluster'
+  kata_ns ||= "openshift-sandboxed-containers-operator"
+  kata_config_name = "example-kataconfig"
+  project(kata_ns)
+  namespace_exists = namespace(kata_ns).exists?
+  kataconfig_exists = kata_config('example-kataconfig').exists?
+  @result_pods = admin.cli_exec(:get, resource: "pods", n:kata_ns, o: "jsonpath='{.items[?(@.spec.runtimeClassName==\"kata\")].metadata.name}'")
+  #@pods_json = @result_pods[:stdout]
+  @error_message = {}
+  node_cmd = "cat /proc/sys/crypto/fips_enabled"
+  #@randomWorkerNodeName = ""
+  @version = admin.cli_exec(:get, resource: "clusterversion", n:kata_ns, o: "jsonpath='{.items[0].status.history[0].version}'")
+  @datajs = admin.cli_exec(:get, resource: "nodes", n:kata_ns, o: "json")
+  @data = JSON.parse(@datajs[:stdout])
+  @nodeAmmount = @data['items'].count
+  @workerNodesAmmount = 0
+  @masterNodesAmmount = 0
+  @data['items'].each do |node|
+    if node['metadata']['name'].include? 'master'
+      @masterNodesAmmount += 1
+    else
+      #@randomWorkerNodeName = node['metadata']['name']
+      @workerNodesAmmount += 1
+    end
+  end
+  @fips_status = node('valiev-kata-49-demo-8lhcq-worker-a-sjz94').host.exec_admin(node_cmd)
```

Comment:
@valtair1891 hardcoded node name.  This won't work for environments that doesn't match your node name.